client/payment-request/style.scss

```diff
@@ -1,4 +1,23 @@
-/* Payment Request Button styles for branded and custom types */
+// Responsive Thickbox wrapper for redirect confirmation dialog.
+.TB_wrapper {
+	position: fixed;
+	display: flex;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
+	z-index: 100050;
+	justify-content: center;
+	align-items: center;
+	pointer-events: none;
+	// Allow pointer events in inner window only, so the overlay
+	// is still clickable and allows for closing the dialog.
+	div {
```

Comment:
maybe safer than this is `> *` but this one works too

Reply:
I actually meant to be more specific about the selector here. Fixed in 877e92b7cde17bb172fcb8737ba37c0a651eac6d.